packages/ng/qualification/src/lib/select/input/qualification-select-input.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Overlay } from '@angular/cdk/overlay';
+import { AfterViewInit, ChangeDetectionStrategy, ChangeDetectorRef, Component, ElementRef, forwardRef, Inject, Input, Renderer2, ViewContainerRef } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { ILuOptionPickerPanel, LuOptionComparer } from '@lucca-front/ng/option';
+import { ILuInputWithPicker } from '@lucca-front/ng/picker';
+import { ALuSelectInputComponent } from '@lucca-front/ng/select';
+import { ILuQualification } from '../../qualification.model';
+import { LuQualificationSelectInputIntl } from './qualification-select-input.intl';
+import { ILuQualificationSelectInputLabel } from './qualification-select-input.translate';
+
+@Component({
+	selector: 'lu-qualification-select',
+	templateUrl: './qualification-select-input.component.html',
+	styleUrls: ['./qualification-select-input.component.scss'],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	providers: [
+		{
+			provide: NG_VALUE_ACCESSOR,
+			useExisting: forwardRef(() => LuQualificationSelectInputComponent),
+			multi: true,
+		}
+	],
+})
+export class LuQualificationSelectInputComponent<D extends ILuQualification = ILuQualification, P extends ILuOptionPickerPanel<D> = ILuOptionPickerPanel<D>>
+	extends ALuSelectInputComponent<D, P>
+	implements ControlValueAccessor, ILuInputWithPicker<D>, AfterViewInit {
+
+	byId: LuOptionComparer<D> = (option1: D, option2: D) => option1 && option2 && option1.id === option2.id;
+
+	@Input() filters: string[];
+	public get allFilters(): string[] {
+		return this.filters == null ? ['sort=job.name,level.position'] :
+			[...this.filters, 'sort=job.name,level.position'];
```

Comment:
`job.name,level.position` is it not the default sort order ?

Reply:
`jobQualifications `are sorted by `id` by default

Reply:
default sort by id = poop

Reply:
for any ressources